changelog.d/2777.misc.rst

```diff
@@ -0,0 +1 @@
+Build does not fail fast when ``use_2to3`` is supplied but set to a false value.
```

Comment:
I think, this should be a bugfix, not a misc change.

Reply:
Bugfix is misc. Naming is weird.

Reply:
Ah, I see. The towncrier config in this repo doesn't declare a separate bugfix type. Many other projects do so. This is what confused me :)